website/content/docs/concepts/workload-identity.mdx

```diff
@@ -6,11 +6,16 @@ description: Learn about Nomad's workload identity feature
 
 # Workload Identity
 
-Every workload running in Nomad is given an identity. When an [allocation][] is
-accepted by the [plan applier][], the leader generates a Workload Identity for
-each task in the allocation. This workload identity is a [JSON Web Token
-(JWT)][] that has been signed by the leader's keyring. The workload identity
-includes the following identity claims:
+Every workload running in Nomad is given a default identity. When an
+[allocation][] is accepted by the [plan applier][], the leader generates a
+Workload Identity for each task in the allocation. This workload identity is a
```

Comment:
Can one allocation have multiple tasks?

Reply:
Yup! A group generates one or more allocations (depending on its `count` value), and since each `group` can have one or more tasks, the allocation can also have multiple tasks.